src/web/theme/ui/molecules/IconWithLabel/IconWithLabel.js

```diff
@@ -3,18 +3,28 @@ import PropTypes from "prop-types";
 import Icon from "theme/ui/atoms/Icon";
 import "./IconWithLabel.scss";
 
-const IconWithLabel = ({ icon, children }) => {
-  return (
-    <span className="icon-with-label">
-      <Icon icon={icon} />
-      {children}
-    </span>
-  );
+const IconWithLabel = ({ icon, children, type = "up" }) => {
+  if (type === "up") {
+    return (
+      <span className="icon-with-label">
+        <Icon icon={icon} />
+        <span>{children}</span>
+      </span>
+    );
+  } else if (type === "down") {
```

Comment:
The "else if" statement is unneeded here, if you return nothing the component will throw